### PR TITLE
BIB-569: Added handling of 2of5 barcodes

### PR DESCRIPTION
**Pull request by tuj**

### CODE REVIEW

File: engine/plugins/state_machine/action_handler.js [hunk 1]
```diff
@@ -247,6 +249,36 @@ class ActionHandler {
         });
     }
 
+    /**
+     * Get item from client.actionData.
+     *
+     * Applies 2of5 transformation of itemIdentifier.
+     *
+     * @param client
+     *
```
Comment (cableman) [inline engine/plugins/state_machine/action_handler.js:258]: Missing parameter and return descripions?

File: engine/plugins/state_machine/action_handler.js [hunk 2]
```diff
@@ -247,6 +249,36 @@ class ActionHandler {
...
+     * @returns {null}
+     */
+    getItem(client) {
```
Comment (cableman) [inline engine/plugins/state_machine/action_handler.js:261]: Should `getItem` be `getIdentifier` og `filterBarcode` ?

**Review by cableman — CHANGES_REQUESTED** [review]

File: engine/plugins/state_machine/action_handler.js (hunk 2 above)
Comment (tuj) [reply engine/plugins/state_machine/action_handler.js:261]: Yes

File: engine/plugins/state_machine/action_handler.js (hunk 1 above)
Comment (tuj) [reply engine/plugins/state_machine/action_handler.js:259]: Fixed

**Review by cableman — APPROVED** [review]

**Pull request merged**